jms/artemis/src/test/resources/amq.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: amq-broker
+  type: Opaque
+  stringData:
+    username: quarkus
+    password: quarkus
+    queues: prices
+
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: amq-broker
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: registry.access.redhat.com/amq-broker-7/amq-broker-72-openshift
```

Comment:
Is this public or behind the wall?

if it is behind the wall you would need `@OnlyIfConfigured("ts.authenticated-registry")`

Reply:
This one is public. Starting with 7.3, it's behind the wall, that's why I stayed with 7.2.

Reply:
(I have actually provided this file to Michal.)